hwilib/cli.py

```diff
@@ -95,7 +95,9 @@ def process_commands(args):
     getkeypool_parser.set_defaults(func=getkeypool_handler)
 
     displayaddr_parser = subparsers.add_parser('displayaddress', help='Display an address')
-    displayaddr_parser.add_argument('path', help='The BIP 32 derivation path of the key embedded in the address')
+    group = displayaddr_parser.add_mutually_exclusive_group()
```

Comment:
Can you make this required? E.g. `group = displayaddr_parser.add_mutually_exclusive_group(required=True)`